lisa/tools/lsmod.py

```diff
@@ -28,6 +30,22 @@ def _check_exists(self) -> bool:
     def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self._command = "lsmod"
 
+    def get_unloaded_modules(self, modules: Union[str, List[str]]) -> List[str]:
+        if isinstance(modules, str):
+            modules_list = modules.split(" ")
+        else:
+            modules_list = modules
+        assert_that(modules_list).described_as(
```

Comment:
"modules shouldn't be empty, when calling get_unloaded_modules."

Reply:
updated